ush/getdump.sh

```diff
@@ -4,35 +4,34 @@ COMPONENT=${COMPONENT:-atmos}
 
 CDATE=${1:-""}
 RUN=${2:-""}
-SOURCE_DIR=${3:-$DMPDIR/${RUN}${DUMP_SUFFIX}.${PDY}/${cyc}/${COMPONENT}}
-TARGET_DIR=${4:-$ROTDIR/${RUN}.${PDY}/${cyc}/${COMPONENT}}
+SOURCE_DIR=${3:-${DMPDIR}/${RUN}${DUMP_SUFFIX}.${PDY}/${cyc}/${COMPONENT}}
+TARGET_DIR=${4:-${ROTDIR}/${RUN}.${PDY}/${cyc}/${COMPONENT}}
 
 DUMP_SUFFIX=${DUMP_SUFFIX:-""}
 
 # Exit if SORUCE_DIR does not exist
-if [ ! -s $SOURCE_DIR ]; then 
-   echo "***ERROR*** DUMP SOURCE_DIR=$SOURCE_DIR does not exist"
+if [ ! -s "${SOURCE_DIR}" ]; then
+   echo "***ERROR*** DUMP SOURCE_DIR=${SOURCE_DIR} does not exist"
    exit 99
 fi
    
-
 # Create TARGET_DIR if is does not exist
-if [ ! -s $TARGET_DIR ]; then mkdir -p $TARGET_DIR ;fi
-
+if [ ! -s "${TARGET_DIR}" ]; then
+   mkdir -p "${TARGET_DIR}"
+fi
 
 # Set file prefix
-cyc=$(echo $CDATE |cut -c 9-10)
-prefix="$RUN.t${cyc}z."
-
+cyc=$(echo "${CDATE}" |cut -c 9-10)
+prefix="${RUN}.t${cyc}z."
 
 # Link dump files from SOURCE_DIR to TARGET_DIR
-cd $SOURCE_DIR
-if [ -s ${prefix}updated.status.tm00.bufr_d ]; then
+cd "${SOURCE_DIR}"
+if [ -s "${prefix}updated.status.tm00.bufr_d" ]; then
```

Comment:
Prefer double braces:
```suggestion
if [[ -s "${prefix}updated.status.tm00.bufr_d" ]]; then
```

Reply:
Thought I'd gotten those, thanks! Committed @ e88b3022b.